packages/lesswrong/server/voteServer.ts

```diff
@@ -223,6 +224,8 @@ export const performVoteServer = async ({ documentId, document, voteType, extend
   const collectionVoteType = `${collectionName.toLowerCase()}.${voteType}`
 
   if (!user) throw new Error("Error casting vote: Not logged in.");
+  
+  if (isLW && user.karma < lowKarmaUserVotingCutoffKarmaSetting.get()) throw new Error(`Error casting vote: You must have ${lowKarmaUserVotingCutoffKarmaSetting.get()}+ karma to vote.`);
```

Comment:
You don't need to add this check, this is already checked in `userCanVote` (on the line below).  In addition to returning `fail` it also returns a `reason: string` if `fail: true`, so you can throw whatever reason it returns as the error.

Reply:
Ah, good point. I originally had the throw error in there (not using the returned reason), but then though mistakenly it was cleaner to put it above